pkg/option/config.go

```diff
@@ -1231,7 +1231,17 @@ func LogRegisteredOptions(entry *logrus.Entry) {
 	}
 	sort.Strings(keys)
 	for _, k := range keys {
-		entry.Infof("  --%s='%s'", k, viper.GetString(k))
+		msg := viper.GetString(k)
+		if len(msg) > 0 {
+			entry.Infof("  --%s='%s'", k, msg)
+		} else {
+			msgs := viper.GetStringSlice(k)
+			if len(msgs) < 0 {
+				entry.Infof("Couldn't get the value flag '--%s'", k)
+				continue
+			}
+			entry.Infof("  --%s='%v'", k, msgs)
```

Comment:
This will print the option in the format `--option=[value1 value2 value3]` due to the `%v` verb being used. It would be nice to get a comma-separated list here so it matches the way the option was passed on the command line.